spec/factories/campaigns.rb

```diff
@@ -19,10 +19,19 @@
 FactoryGirl.define do
   factory(:campaign) do
     volunteer_center
-    name 'MyString'
-    description 'MyString'
-    donation_dateline Date.today + 7
+    name 'Share Your Christmas 2015'
```

Comment:
I like to leave the defaults non-sensical (e.g. use MyString as the default, which is the default when FG generates them), to make it clear that they don't matter. I would typically override them by using a trait (see below) or inline like this:

`let(:xmas) { FactoryGirl.create :campaign, name: 'Christmas 2016' }`

These are OK as you've changed them but if you're doing debugging of your tests you could end up with an "accidental" pass or failure because you've set meaningful defaults inside the factory (makes it hard to determine that something was wrong)